docs/howto/operator.rst

```diff
@@ -1386,13 +1392,30 @@ Some arguments in the example DAG are taken from OS environment variables:
     :start-after: [START howto_operator_cloudsql_arguments]
     :end-before: [END howto_operator_cloudsql_arguments]
 
-Example body defining the instance:
+Some other arguments are created based on the arguments above:
+
+.. literalinclude:: ../../airflow/contrib/example_dags/example_gcp_sql.py
+    :language: python
+    :start-after: [START howto_operator_cloudsql_create_arguments]
+    :end-before: [END howto_operator_cloudsql_create_arguments]
+
+Example body defining the instance with failover replica:
 
 .. literalinclude:: ../../airflow/contrib/example_dags/example_gcp_sql.py
     :language: python
     :start-after: [START howto_operator_cloudsql_create_body]
     :end-before: [END howto_operator_cloudsql_create_body]
 
+Example body defining read replica for the instance above:
+
+.. literalinclude:: ../../airflow/contrib/example_dags/example_gcp_sql.py
+    :language: python
+    :start-after: [START howto_operator_cloudsql_create_replica]
+    :end-before: [END howto_operator_cloudsql_create_replica]
+
+Note: Failover replicas are created together with the instance in a single task.
+Read replicas need to be created in separate tasks.
+
```

Comment:
I'd add example usages of the create / delete operator to create and delete read / failover replicas from the example DAG somewhere here.

Reply:
Example of creating failover and read replicas is documented few lines above that notes :)
Those notes refer to those examples.
However I will add deletion examples to documentation (in the CloudSqlInstanceDeleteOperator section)

Reply:
Examples added

Reply:
Ok for the delete example. However, for the create, I'd still add a separate code example here for creating the read replica (under "Using the operator").